documentation/api/query/v4/operators.markdown

```diff
@@ -203,6 +203,58 @@ When querying a particular endpoint, such as `/pdb/query/v4/nodes`, the endpoint
 the context for the query. Querying the [root] endpoint requires specifying a
 context explicitly.
 
+## Paging operators (`limit`, `offset`, `order_by`)
+
+PuppetDB allows specification of paging clauses within a "from" clause in a
+query or subquery. The `limit` and `offset` operators both accept an
```

Comment:
I proposed a rename of the doc actually in my PQL patch to 'AST query language' and making it more about the AST language, instead of just about operators and such. I'd be happy to rethink our nomenclature.